### PR TITLE
feat: migrate to go modules

### PR DESCRIPTION
**Pull request by schmidtp0740**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,11 +19,10 @@ install:
   - python3 -m pip install -U pip
   - pip3 install --upgrade "git+https://github.com/ibm/detect-secrets.git@master#egg=detect-secrets"
   - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin
-  - go get -u github.com/kardianos/govendor
 
 before_script:
   - bin/detect_secrets.sh
-  - govendor sync
+  - go mod vendor
```
Comment (jorge-ibm) [inline .travis.yml:25]: Now that this is changing, can you also please update the `README`?

**Review by Aerex — COMMENTED** [review]
Tests well on my end :+1:  I was able to run some bdd tests against the CLI using this branch of the sdk.

Nit: Can we update the README with new instructions on how to build and install the sdk?

**Review by jorge-ibm — CHANGES_REQUESTED** [review]
Just one doc request but the migration looks good! Pulled this version into the CLI and didn't run into any issues. Thanks for doing this!

File: .travis.yml (hunk 1 above)
Comment (Aerex) [reply .travis.yml:25]: I wish there was a jinkx emoticon lol

**Review by jorge-ibm — APPROVED** [review]

**Pull request merged**